2022/02/ChAoS-UnItY/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2022 ChAoS-UnItY
+
```

Comment:
Could you figure out a way to symlink to license file instead of duplicating it, simliar to what [this PR has done](https://github.com/rust-tw/advent-of-code/commit/7ad20fcc0b0d96b60181b67068622fba40292c6f)?

Reply:
Sure, I'll follow Pan's workaround, thanks for mentioning that.